.cbmc-batch/jobs/aws_hash_table_init/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+#size 8 takes 41s
+#size 16 takes 3m1.125s
+MAX_TABLE_SIZE ?= 8
+DEFINES += -DMAX_TABLE_SIZE=$(MAX_TABLE_SIZE) 
+
+#A hash entry is 24 bytes, which is 3 iters. So we need 3 * MAX_TABLE_SIZE + 1
+UNWINDSET += memset_override_0_impl.0:$(shell echo $$(($$((3 * $(MAX_TABLE_SIZE))) + 1)))
+
+#CBMCFLAGS += --remove-function-body s_default_malloc
+
+DEFINES += -DCBMC
```

Comment:
Just noting that another PR did `-DCBMC=1`. The above suggests to use `#ifdef CBMC`, the other PR suggests using `#if CBMC == 1`

Reply:
Which do you suggest?

Reply:
I'm agnostic: is there a best practice?

Reply:
I'd use `-DCBMC` as you really only care about it being defined/undefined, not whether it's 1, 2, etc.